File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 4/ex1.c
#include <stdio.h>

main()
{
    printf("Soma de Dois Elementos\n\n");

    //vars
    int v[12], i, x, y, soma;

    //Entradas
    for( i=0; i<12; i++)
    {
        printf("Digite v[%d]: ", i);
        scanf("%d",&v[i]);
    }

    printf("\nDigite uma posicao X(0<=X<12): ");
    scanf("%d",&x);

    printf("Digite uma posicao Y(0<=Y<12): ");
    scanf("%d", &y);

    //Processamento
    soma = v[x] + v[y];

    //Saída
    printf("Soma = %d + %d = %d", v[x], v[y] soma);


}

File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 4/src/Ex1.java
//Importando Scanner
import java.util.Scanner;

public class Ex1 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Soma de dois valores de um vetor\n\n");
		
		//vars
		int v[] = new int[12];
		int i, x, y, soma;
		
		//Entradas
		for( i=0; i<12; i++)
		{
			System.out.print("Digite v[" + i +"]: ");
			v[i] = ent.nextInt();
		}
		
		//Entrada
		System.out.println("\nDigite uma posição X (0 <= X < 12): ");
		x = ent.nextInt();
		System.out.println("Digite uma posição Y (0 <= Y < 12): ");
		y = ent.nextInt();
		
		//Processamento
		soma = v[x] + v[y];
		
		//Saída
		System.out.println("Soma: " + soma);
		
		
	}
}
